integrate/romberg_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integrate
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func TestRomberg(t *testing.T) {
```

Comment:
Can you make this table-driven please.

Reply:
Tests are now on the same format as **trapezoidal_test**